documentation/extracts.md

```diff
@@ -0,0 +1,61 @@
+
+### Case Type
+Case type extracts cover the extraction of data for cases which sit underneath a specific case type.
+
+The available export types and purposes:
+
+| Type           | Purpose                                                                                                                                                           |
+|----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| CASE_DATA      | Provides a flat view of case data                                                                                                                                 |
+| ALLOCATIONS    | Provides a view of every time a case has been allocated and unallocated, as well as the entry and exit times for each stage and also the deadlines for each stage |
+| TOPICS         | Provides all the topics added to a case by date range                                                                                                             |
+| CORRESPONDENTS | Link the UUIDS from case data to correspondents                                                                                                                   |
+| CASE_NOTES     | Provides the case notes added to a case by date range                                                                                                             |
+| EXTENSIONS     | Provides details on all extensions applied to a case                                                                                                              |
+| APPEALS        | Provides details on all appeals added to a case                                                                                                                   |
+| INTERESTS      | Provides details on all interest from internal and external sources recorded against a case                                                                       |
+| SUSPENSIONS    | Provides details on each time a case has had a suspension applied or removed                                                                                      |
+
+### Environment 
+The available environment export types are:
+
+| Type        | Purpose                                                                  |
+|-------------|--------------------------------------------------------------------------|
+| Topics      | Provides a list of topics in the database, with matching UUIDS           |
+| Teams       | Provides a list of teams in the database, with matching UUIDS            |
+| Units/teams | Provides a list of units and their associated teams, with matching UUIDS |
+| Users       | Provides a list of users in the database, with matching UUIDS            |
+
+### Single Object, Multiple Use (SOMU)
+Only some combinations of case types and SOMU types result in valid extract URLs, as not all case types use SOMU functionality.
```

Comment:
Should this have a table of SOMU types - possiblty also listing which case types they apply to?

Reply:
I wasn't sure whether to go to that level of detail in the documentation as I see that more as configuration rather than part of the feature. I could maybe add that somu types are defined in the info schema so an engineer to go and look there for a defined configuration?

Reply:
Sounds good - I think all of my comments apart from the url one are tied to a use case for this doc where "I need to export X specific report - Is that possible and if so, how do I do that" so pointing to where to reference the config they need will work for that.

Reply:
Made some additions